Makefile

```diff
@@ -56,7 +56,9 @@ prefix_install: installables
 	mkdir -p "$(PREFIX)/Frameworks" "$(PREFIX)/bin"
 	rsync -a --delete "$(TEMPORARY_FOLDER)$(FRAMEWORKS_FOLDER)/$(OUTPUT_FRAMEWORK)" "$(PREFIX)/Frameworks/"
 	cp -f "$(TEMPORARY_FOLDER)$(BINARIES_FOLDER)/carthage" "$(PREFIX)/bin/"
-	install_name_tool -rpath "/Library/Frameworks" "@executable_path/../Frameworks/$(OUTPUT_FRAMEWORK)/Versions/Current/Frameworks/"  "$(PREFIX)/bin/carthage"
+	install_name_tool -delete_rpath "@executable_path/../Frameworks" "$(PREFIX)/bin/carthage" # Avoid duplication of "@executable_path/../Frameworks"
+	install_name_tool -rpath "/Library/Frameworks" "@executable_path/../Frameworks" "$(PREFIX)/bin/carthage"
```

Comment:
I confirmed that `carthage` had following `rpath` by adding the change on xcodeproj to this PR:
```
Load command 36
          cmd LC_RPATH
      cmdsize 32
         path @executable_path/. (offset 12)
Load command 37
          cmd LC_RPATH
      cmdsize 80
         path @executable_path/CarthageKit.framework/Versions/Current/Frameworks (offset 12)
Load command 38
          cmd LC_RPATH
      cmdsize 96
         path @executable_path/../Frameworks/CarthageKit.framework/Versions/Current/Frameworks (offset 12)
Load command 39
          cmd LC_RPATH
      cmdsize 48
         path @executable_path/../Frameworks (offset 12)
Load command 40
          cmd LC_RPATH
      cmdsize 40
         path @loader_path/../Frameworks (offset 12)
```
:+1: